src/api/members/members.dto.ts

```diff
@@ -32,13 +32,7 @@ export class MemberDto implements IMember {
     @IsOptional()
     public joinedOn?: Date;
 
-    @ApiPropertyOptional()
-    @IsOptional()
-    @IsInt()
-    public access?: number;
-
-    @ApiPropertyOptional()
-    @IsOptional()
+    @ApiProperty()
     @IsInt()
     public sid?: number;
```

Comment:
Late to the party, but if this is non-optional, should this still be optional?

Reply:
I was sure I did this and something still needed it to be able to be undefined and broken somewhere, but I have no idea what that could be, and I've just done it and everything seems to work fine, so I might make a PR that makes this change and also updates all the packages I updated (hopefully closing those depend-a-bot PRs) that I did when installing this on a new laptop to try to run it again... (I realise that this isn't exactly my jurisdiction either anymore... but I guess I did make this PR).